### ISSUE TITLE
No git in ubuntu/debian images

### ISSUE THREAD

**Issue by rvdh** [William-Yeh/docker-ansible#33]
It's currently not possible to use ansible-galaxy with roles from git, like this:

```
- src: ssh://git@github.com/rvdh/some-ansible-role.git
  scm: git
```
Right now I fixed this by adding git to the packages in the dockerfile, but maybe this can be fixed here?

**Comment by rvdh** [comment]
Never mind, just found https://github.com/William-Yeh/docker-ansible/issues/20

**Comment by rvdh** [comment]
Actually, it seems like the fix mentioned in #20 was not applied to Debian 9 / Ubuntu 16.x versions..
